### PR TITLE
Add flake8 and isort in docs

### LINKED ISSUE #1271: Flake8 chapter in docs
As we enforce currently Flake8 as code formatting, I think there should be a bit of information on why we chose flake 8 and how it can be applied (e.g. a recommended IDE plugin for pycharm or VS Code) and that it is also now part of the pre-commit.
At the moment, the docs do not mention it at all :https://docs.manim.community/en/latest/contributing/development.html#for-first-time-contributors
EDIT:
here are two good video resources that could be embedded into the docs:

For the general purpose of flake8:
https://www.youtube.com/watch?v=TDUf93vqq3g
For VS Code:
https://www.youtube.com/watch?v=eMIxokGhFHM
I actually don't know how to copyright situation of embedding youtube videos on websites is.

### PR DESCRIPTION
**Pull request by Nikhil0504**
## Changelog / Overview

added 'flake8' and 'isort' usages to docs

## Motivation

Now people can know what type of formatting we are using and how they can use.
## Explanation for Changes

People don't need to search the internet for the usage of these.
## Further Comments

Closes #1271 
## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

**Comment by Darylgolden** [comment]
LGTM

File: docs/source/contributing/development.rst [hunk 1]
```diff
@@ -195,11 +195,13 @@ that manim still works as intended and that the code you added
 sticks to our coding conventions.
 
 - *Code style*: We use the code style imposed
-  by `Black <https://black.readthedocs.io/en/stable/>`_. The pipeline
+  by `Black <https://black.readthedocs.io/en/stable/>`_, `isort <https://pycqa.github.io/isort/>`_
+  or `flake8 <https://flake8.pycqa.org/en/latest/>`_. The pipeline
```
Comment (kolibril13) [inline docs/source/contributing/development.rst:199]:
```suggestion
  by `Black <https://black.readthedocs.io/en/stable/>`_, `isort <https://pycqa.github.io/isort/>`_ and  `flake8 <https://flake8.pycqa.org/en/latest/>`_. The pipeline
```

**Comment by kolibril13** [comment]
As we use all three of them, it is not or, but and

**Comment by Nikhil0504** [comment]
> As we use all three of them, it is not or, but and

Changed it

File: docs/source/contributing/development.rst [hunk 2]
```diff
@@ -195,11 +195,13 @@ that manim still works as intended and that the code you added
...
 
 - *Code style*: We use the code style imposed
-  by `Black <https://black.readthedocs.io/en/stable/>`_. The pipeline
+  by `Black <https://black.readthedocs.io/en/stable/>`_, `isort <https://pycqa.github.io/isort/>`_
+  and `flake8 <https://flake8.pycqa.org/en/latest/>`_. The pipeline
   makes sure that the (Python) files changed in your pull request
   also adhere to this code style. If this step of the pipeline fails,
-  fix your code style by running ``black <file or directory>`` to
-  automatically format your files.
+  fix your code style by running ``black <file or directory>`` or
```
Comment (kolibril13) [inline docs/source/contributing/development.rst:202]: here as well

Comment (Nikhil0504) [reply docs/source/contributing/development.rst]: Done

File: docs/source/contributing/development.rst [hunk 3]
```diff
@@ -195,11 +195,13 @@ that manim still works as intended and that the code you added
...
-  by `Black <https://black.readthedocs.io/en/stable/>`_. The pipeline
+  by `Black <https://black.readthedocs.io/en/stable/>`_, `isort <https://pycqa.github.io/isort/>`_
+  and `flake8 <https://flake8.pycqa.org/en/latest/>`_. The pipeline
   makes sure that the (Python) files changed in your pull request
   also adhere to this code style. If this step of the pipeline fails,
-  fix your code style by running ``black <file or directory>`` to
-  automatically format your files.
+  fix your code style by running ``black <file or directory>`` and 
+  ``isort <file or directory>`` to automatically format your files. You could also use `flake8` 
+  if you want to modify the style manually by running ``flake8 <file or directory>``.
```
Comment (kolibril13) [inline docs/source/contributing/development.rst:204]:
```suggestion
To fix code style problems, run ``flake8 <file or directory>`` for a style report, and then fix the problems that were detected by ``flake8 ``
```

Comment (kolibril13) [reply docs/source/contributing/development.rst:204]: I think the tone here can have a bit more certainty.

Comment (Nikhil0504) [reply docs/source/contributing/development.rst:204]: Ya this works too

**Review by jsonvillanueva — APPROVED** [review]

**Pull request merged**